assets/stylesheets/modules/_roundel.scss

```diff
@@ -0,0 +1,68 @@
+/* ==========================================================================
+   Roundel
+   ========================================================================== */
+
+.roundel {
+    position:relative;
+}
+
+.roundel--raised {
+    top: -53px;
+
+    @include mq(desktop) {
+        top: -58px;
+    }
+}
+
+
+.roundel__container {
+    @include circular();
+    position: absolute;
+    z-index: 2;
+    right: 0;
+    text-align: center;
+    background: rgb(150,215,42);
+    width: 110px;
+    height: 110px;
+    padding: 10px 10px;
+    margin: 10px 10px;
+
+    @include mq(desktop) {
+        width: 140px;
+        height: 140px;
+        padding: 20px 20px;
+        margin: 20px 20px;
+    }
+
+    align-content: center;
```

Comment:
These should go before the media query - media queries should always override more general rules above them